docs/source/requests.md

```diff
@@ -1469,6 +1475,192 @@ Let's consider an example of changing a value of a NODE transaction's `service`
 ```
 
 
+### AUTH_RULES
+
+A command to set multiple AUTH_RULEs by one transaction. 
+Transaction AUTH_RULES is not divided into two AUTH_RULE transaction, and is written to the ledger with one transaction with the full set of rules that come in the request.
```

Comment:
What does `is not divided into two AUTH_RULE transaction` mean?

Reply:
Updated this description.